articles/inter-vlan-routing/index.md

```diff
@@ -0,0 +1,160 @@
+
+VLANs make it easier to segment a workstation into different isolated LANs. They improve the performance of the network and are flexible, since they are logically connected.
+
+VLANs act as separate subnets on the network. To move packets from one VLAN to another and enable communications between hosts, we have to configure the network to enable inter-VLAN routing.
+
+### Inter-VLAN routing methods
+#### Legacy inter-VLAN routing
+In this method, multiple router interfaces are used, each connecting to a switch port in different VLANs. These interfaces are served as default gateways. Sometimes, it may lead to network issues, since expanding the network requires additional cabling.
+
+#### Router-on-a-stick
+In this method, unlike the legacy routing, one physical interface port is used for routing the traffic between the network segments. The network administrator need not create separate VLAN interfaces like `fa0/1` to `fa0/10`. Instead, all the interfaces from 1 to 10 are created with a single interface.
+
+#### Layer 3 switch using Switched Virtual Interface (SVI)
+Currently, this method of inter-VLAN routing that uses layer 3/multilayer switch and Switched Virtual Interfaces (SVI) are being used more commonly. SVIs are created for VLANs that exist on the switch and they perform the same function for the VLANs as that of a router.
+
+Layer 3 switches are expensive, hence not affordable.
+
+### Network configurations for Inter-VLAN communication using Router-On-Stick method
+In this article, we will learn how to configure inter-VLAN routing using the router-on-a-stick method.
+
+Consider a LAN with 4 PCs, 1 switch, and a router connected as shown in the image:
+
+![Local Area Network](/engineering-education/inter-vlan-routing/network.jpg)
+
+We want to configure two VLANs 10 and 20. PC0 and PC1 on VLAN10 while PC2 and PC3 are on VLAN20.
+* IP Address of PC0   -192.168.1.10
+* IP Address of PC1   -192.168.1.20
+* IP Address of PC2   -192.168.2.10
+* IP Address of PC3   -192.168.2.20
+* The default gateway for VLAN10- 192.168.1.1
+* The default gateway for VLAN10- 192.168.2.1
+
+#### Step 1
+For us to subdivide the network into two subnets, we have to create two VLANS on the switch,VLAN10 and VLAN20. Give them custom names. That is VLAN 10 -student and VLAN 20 - staff.
+
+*To create two vlans we enter the configuration mode using **`config terminal`** command then we enter the vlan number like vlan 10 then the name and then exit*
+```
+Switch>enable           //moving from user exec mode to priviledge mode
+Switch#config terminal    //moving from priviledge mode to global configuration mode
+Switch(config)#vlan 10    //assigning vlan number
+Switch(config-vlan)#name student   //assigns the vlan 10 the name student
+Switch(config-vlan)#exit
+Switch(config)#vlan 20
+Switch(config-vlan)#name staff   //assigns vlan 20 the name staff
+Switch(config-vlan)#exit
```

Comment:
I doubt if the comments can be added this way. When I read through a few articles, for adding comments, you must use `!` instead of `//`

Reply:
Make these changes wherever necessary